resources/lang/en/Config.properties

```diff
@@ -1,9 +1,4 @@
-FeedbackSettingsTab=Feedback settings
-releaseFeedbacksLabel=Release feedbacks automatically
-timestampVisibilityLabel=Show date for the feedback
-allowNoRatingFeedbackLabel=Allow feedbacks with no rating
-allowFeedbacksOnlyIfPurchasedLabel=Allow adding feedbacks only if the item was purchased
-showEmptyRatingsInCategoryViewLabel=Show empty ratings in category view
-numberOfFeedbacksLabel=Maximum number of feedbacks per author and item
+FeedbackSettingsTab=Settings
+showEmptyRatingsInCategoryViewLabel=Show empty star ratings in the category view 
 showRatingFacetLabel=Show rating filter
-showRatingSortingLabel= Show sorting option in item lists
\ No newline at end of file
+showRatingSortingLabel=Show sorting option in item lists
```

Comment:
Ist es egal ob die values in "" stehen?

Reply:
Grundsätzlich ja, sollte aber innerhalb eines Projekts einheitlich sein
@felixgehrmann

Reply:
done